### PR TITLE
Migrate program-level stuff to ProgramRituals

### PR DESCRIPTION
**Pull request by mogul**

### CODE REVIEW

File: DeliveryProcess.md [hunk 1]
```diff
@@ -32,132 +31,22 @@ cloud.gov's business unit (BU) focuses on business strategy and development, ens
...
 
-- Funnel (to note that we have heard about the need but otherwise not advanced it)
-- Analysis (where we explore our options for overall UX and implementation, and set objective metrics and outcomes)
-- Backlog (when the feature is basically well-understood at a high level and awaits squad-level attention)
-- Implementing (while implementation is in progress)
-- Verifying (while they are being evaluated as delivering their intended benefit, open to customer feedback)
-- Done (when they are confirmed as meeting their target, and no longer a focus)
+Cards on our boards typically capture and track a bug, raw task, or Story. Bugs and tasks are what they sound like. **Stories** represent tactical increments of individually-valuable work deliverable by a squad within a single iteration... often an isolated change in functionality aimed at achieving a goal for a particular kind of stakeholder, whether customer, user, or operator/admin. We try to set the scope of stories such that we anticipate they will take no more than three days of concentrated work to complete. A Story gets tracked on the Favro collection of a squad once that squad explicitly takes responsibility for it.
```
Comment (mogul) [inline DeliveryProcess.md]: This story definition should probably move right to the top of the Story Lifecycle doc.

File: DeliveryProcess.md [hunk 2]
```diff
@@ -32,132 +31,22 @@ cloud.gov's business unit (BU) focuses on business strategy and development, ens
...
-Features are also tracked against the Program Increment (PI) where we suspect they'll be delivered based on our best understanding of our current priorities and progress.
+[Lifecycle and "definition-of-done" for stories](#)
 
-**Stories** progress through these columns (although there is some variation from squad to squad):
+### Cards on program boards
 
-- Triage (where reported issues, and newly-identified stories appear)
-- Backlog (when they are sequenced alongside other stories prioritized for attention)
-- Icebox (where they go when they are not planned to get attention any time soon)
+**Features** are user-notable, program-sequenced, roadmap-worthy changes in product capabilities tracked against cadence-driven periodic milestones... big stuff that's worth planning and announcing on a regular basis. Features are fairly general, encompassing a set of stories which break down the actual changes in product behavior that we're after to get the intended benefits of the feature. Generally we do coarse analysis and UX exploration work at the Feature level before breaking them down into stories.
```
Comment (mogul) [inline DeliveryProcess.md]: This feature definition should probably move right to the top of the Feature Lifecycle doc.

**Comment by mogul** [comment]
Tagging @brittag for any changes we need to make in references to these docs from elsewhere that were referring to the compliance bits that moved around.

**Comment by brittag** [comment]
@mogul We have a few important references in the SSP that would break:

> The specific security impact analysis step is in the "grooming" section at https://github.com/18F/cg-product/blob/master/DeliveryProcess.md#grooming

> These tollgates are documented in our Delivery Process: https://github.com/18F/cg-product/blob/master/DeliveryProcess.md (for example, see under “In Progress”: “The deployment must follow our Configuration Management plan. If not possible, a new issue is filed in cg-site to update the plan.”)

> The cloud.gov Delivery Process ( https://github.com/18F/cg-product/blob/master/DeliveryProcess.md#definition-of-done ) also includes requirements along with recommended items.

We shouldn't break those references since they're key for the control narratives that they're in, but I'm not sure the best way to rearrange this change to make sure people find the right information when they're reading the SSP - I'd be happy to look at it together and look at options.

**Comment by brittag** [comment]
Also after this is merged, I want to file some changes to these docs to reflect our current compliance processes, since statements like this are out of date: "If the work changes an aspect of our system or operating environment that is (or would ideally be) documented in our System Security Plan (SSP), a new issue is filed in cg-compliance to note the change."

Noting here as a reminder to myself!

**Pull request merged**